apps/roam/src/utils/supabaseContext.ts

```diff
@@ -5,8 +5,10 @@ import getRoamUrl from "roamjs-components/dom/getRoamUrl";
 import { Database } from "@repo/database/types.gen";
 
 type Platform = Database["public"]["Enums"]["Platform"];
+type roamAlphaAPI = Window["roamAlphaAPI"];
 
 export type SupabaseContext = {
+  api: roamAlphaAPI;
```

Comment:
This shouldn't be needed.  `roamAlphaAPI` is always available on the `window` object